terraform/main.tf

```diff
@@ -38,7 +38,7 @@ locals {
   }
   proxies = {
     for name, instance in local.provisioner_pairs :
-    name => instance
+    instance.prefix => true
```

Comment:
ah cool, so you end up with a map with just a single key if any of them has use_proxy, otherwise you get an empty map?

Reply:
Yes. Just tested. If `use_proxy` is always `false`, no proxy server will be created and as such the map will be empty.